src/kagglehub/auth.py

```diff
@@ -1,2 +1,13 @@
+import json
+from kagglehub.config import set_kaggle_credentials
+
+
 def login():
-    raise NotImplementedError()
+    """Prompt the user for their Kaggle username and API key and save them globally."""
+
+    username = input("Enter your Kaggle username: ")
```

Comment:
This is great for the command-line case.

Does this work in a Jupyter notebook? Should we create a nicer widget like I suggested in the bug for the Jupyter case? Perfectly fine to address in a separate PR.

Reply:
yes it does. maybe lets address the nicer widget part in a separate cl? it should be simple.